cpp/src/Ice/TcpEndpointI.cpp

```diff
@@ -15,7 +15,9 @@
 #    include "TcpAcceptor.h"
 #    include "TcpConnector.h"
 #    include "TcpEndpointI.h"
+
```

Comment:
The new line here seems odd, is only "TcpEndpointI.h" that should be treated specially the orders should stay alphabetically sorted.

Reply:
I removed all these extra newlines.